python/tank/platform/engine.py

```diff
@@ -169,6 +171,13 @@ def __init__(self, tk, context, engine_instance_name, env):
         qt.QtGui = base_def.get("qt_gui")
         qt.TankDialogBase = base_def.get("dialog_base")
 
+        qt5_root = self.__define_qt5()
```

Comment:
Good idea about making the straight import of everything part of the importer. I'll add that.

As for _define_qt_base vs __define_qt5, see my general comment at the bottom of the code review.